### PR TITLE
Move images from redhat-chaos to krkn-chaos quay org

### PR DESCRIPTION
**Pull request by chaitanyaenr**
This commit also updates docs with the new references.

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -4,84 +4,84 @@ services:
...
         context: ./
         dockerfile: ./cerberus/Dockerfile
-      image: quay.io/redhat-chaos/cerberus:kraken-hub
+      image: quay.io/krkn-chaos/cerberus:kraken-hub
     cerberus-prow:
       build:
         context: ./
         dockerfile: ./cerberus/Dockerfile_prow
-      image: quay.io/redhat-chaos/cerberus:prow
+      image: quay.io/krkn-chaos/cerberus:prow
```
Comment (paigerube14) [inline docker-compose.yaml:12]: for now the cerberus repo is still at redhat-chaos

Comment (paigerube14) [reply docker-compose.yaml:12]: let's put in second pr when cerberus is moved over to krkn-chaos

Comment (chaitanyaenr) [reply docker-compose.yaml]: Done, thanks.

**Review by paigerube14 — APPROVED** [review]
Lgtm

**Pull request merged**